code/common/subscriptions.q

```diff
@@ -86,10 +86,15 @@ subscribe:{[tabs;instrs;setschema;replaylog;proc]
 		/-insert the details into the SUBSCRIPTIONS table
 		.lg.o[`subscribe;"subscription successful"];
 		updatesubscriptions[proc;;instrs]each subtabs];
-		/-return the names of the tables that have been subscribed for and
-		/-the date from the name of thr tickerplant log file (assuming the tp log has a name like `: sym2014.01.01
-		/-plus .u.i and .u.icounts if existing on TP - details[1;0] is .u.i, details[2] is .u.icounts (or null)
-		(`subtables`tplogdate!(details[0;;0];(first "D" $ -10 sublist string last details 1)^logdate)),{(where 101 = type each x)_x}(`i`icounts`d)!(details[1;0];details[2];details[3])
+	/-return the names of the tables that have been subscribed for and
+	/-the date from the name of thr tickerplant log file (assuming the tp log has a name like `: sym2014.01.01
+	/-plus .u.i and .u.icounts if existing on TP - details[1;0] is .u.i, details[2] is .u.icounts (or null)
+	if[`tickerplant=proc`proctype;
+		:(`subtables`tplogdate!(details[0;;0];(first "D" $ -10 sublist string last details 1)^logdate)),{(where 101 = type each x)_x}(`i`icounts`d)!(details[1;0];details[2];details[3])];
+	if[`segmentedtp=proc`proctype;
```

Comment:
Given that we have called the other one "tickerplant", can we call this one "segmentedtickerplant"